### ISSUE TITLE
No routes matched location "/_next/webpack-hmr"

### ISSUE THREAD

**Issue by HenkDz** [zabirauf/AutoGPT.js#15]
I kept getting this error, also it says:
"1 critical severity vulnerability
To address all issues, run:
  npm audit fix" 
  I did that and still see the webpack error show in screenshot bellow.
  
![image](https://user-images.githubusercontent.com/17301927/233226716-521bd144-5195-44e4-8522-49bfd8888dc0.png)

**Comment by zabirauf** [comment]
@HenkDz can you share what OS, Node version you are using and the list of commands you are using to start the server?

**Comment by HenkDz** [comment]
OS: Win 11
Node: 18.15.0
NPM: 8.14.0

![image](https://user-images.githubusercontent.com/17301927/233500756-4906c9a3-b4a5-4077-9b5b-0b5db4acbb50.png)

**Comment by HenkDz** [comment]
@zabirauf ?

**Comment by zabirauf** [comment]
@HenkDz I dont have an access to Windows machine at the moment to unable to repro it. From the screenshots looks like something on you machine is calling that route, do you have some app or browser plugin for Next.js development as the path looks related to that. 

However, this shouldn’t impact anything for development and can you confirm that going to localhost:3000 works fine for you?

**Comment by HenkDz** [comment]
Yes the webui works fine, it's just this error filling my console up that's a bit annoying.
